packages/core/admin/ee/server/services/__tests__/stages.test.js

```diff
@@ -64,4 +96,50 @@ describe('Review workflows - Stages service', () => {
       });
     });
   });
+  describe('replaceWorkflowStages', () => {
+    test('Should create a new stage and assign it to workflow', async () => {
+      await stagesService.replaceWorkflowStages(1, [
+        ...workflowMock.stages,
+        {
+          name: 'to publish',
+        },
+      ]);
+
+      expect(servicesMock['admin::workflows'].findById).toBeCalled();
+      expect(entityServiceMock.create).toBeCalled();
+      expect(entityServiceMock.update).not.toBeCalled();
+      expect(entityServiceMock.delete).not.toBeCalled();
+    });
```

Comment:
Do you think it's worth writing an expectation for the result of `replaceWorkflowStages` in these tests?

e.g. here I think it should be = `{ stages: [ 1, 2, 3, 4 ] }`

Reply:
I do not think it is useful to check the result of `replaceWorkflowStages` function as it is the result of `workflowsService.update` and it is mocked.
But surely I can check what is sent to `workflowsService.update` function.

Reply:
Oh yes I see 👍🏻